README.md

```diff
@@ -93,15 +93,56 @@ Once the page is loaded, the browser sends further async requests as needed.
 20. ### What is a Higher Order Component ? Give example and explain the use cases where it should be used.
      Answer To be added
 21. ### How to deep clone a object in javascript ?
-     Answer To be added
+     	(1)Assuming that you only have variables and not have any functions in your object,just use:
+		var NewObject = JSON.parse(JSON.stringify(OldObject));	
+	(2)Copying an object with the Object.assign() method
```

Comment:
Solution 2 and 3 are incorrect. Both Object.assign() and spread operator would do `shallow copy` and not `deep copy`

Reply:
Yes, i have attached all the copy method.